src/tests/Dashboard/reducer/conceptReducer.test.js

```diff
@@ -36,4 +39,29 @@ describe('Test suite for concepts reducer', () => {
       loading: false,
     });
   });
+
+  it('should handle SEARCH_CONCEPTS', () => {
+    expect(reducer(
```

Comment:
You would want to make your it block messages more descriptive when writing tests for reducers, actions and components. Tests are usually the first place where someone onboarding into a codebase can look into and get an idea of what each unit function is doing, and you'll be helping out if your test case message is the actual description of what that function or unit block of code being tested is actually doing..

a better way you could write this could be

```
it sets redux state key loading and hasMore to false
on dispatching action type SEARCH_CONCEPTS.`
```
overall, I think you have done a great Job.

Reply:
I like it so much when the new guys on the team also start reviewing code! 👍

Reply:
Thank you for the feedback @Efosaok. I have made the relevant changes to my tests to make them more descriptive as suggested.
@dkayiwa we like this too. Thanks @Efosaok for commenting